examples/pytorch/ogc/utils.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import scipy.sparse as sp
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+
+
+def sparse_mx_to_torch_sparse_tensor(sparse_mx):
+    """Convert a scipy sparse matrix to a torch sparse tensor."""
+    sparse_mx = sparse_mx.tocoo().astype(np.float32)
+    indices = torch.from_numpy(
+        np.vstack((sparse_mx.row, sparse_mx.col)).astype(np.int64)
+    )
+    values = torch.from_numpy(sparse_mx.data)
+    shape = torch.Size(sparse_mx.shape)
+    return torch.sparse.FloatTensor(indices, values, shape)
+
+
+def symmetric_normalize_adjacency(graph):
+    """Symmetric normalize graph adjacency matrix."""
+    adj = graph.adjacency_matrix()
```

Comment:
Did you try using [dgl.sparse](https://docs.dgl.ai/en/1.1.x/api/python/dgl.sparse_v0.html) directly? Which may save you a lot of code from converting sparse format, and simplify the code for readability.

Reply:
Thank you for the excellent suggestion. By utilizing dgl.sparse, we have  effectively streamlined our normalization-related code in the "utils.py", reducing it from 21 lines to just 7 lines, all within a single function.